### PR TITLE
Starfinder 2e Dark Theme Stylesheet

### PR DESCRIPTION
**Pull request by LiftVortex**
Update:
- I've updated the files and structure to be the same as the newest main commit
- Included a Dark Mode option in the settings toggle to use this theme.
- Update the en.json to include Dark.
- Changed the logo to match the new commit.

Fork includes:
- Stylesheet which customises the PC Sheet.

### CODE REVIEW

**Comment by stwlam** [comment]
Would you mind reducing this down to the PC sheet so it can be considered in isolation?

**Comment by LiftVortex** [comment]
> Would you mind reducing this down to the PC sheet so it can be considered in isolation?

Yeah no problem, I'll split it into two style sheets.

**Comment by LiftVortex** [comment]
> Would you mind reducing this down to the PC sheet so it can be considered in isolation?

I've now updated the pull request to just the PC Sheet.

**Comment by LiftVortex** [comment]
@stwlam I've now updated the files and structure to be the same as the newest main commit which included the light mode option.

Changes below:
- Included a Dark Mode option in the settings toggle to use this theme.
- Update the en.json to include Dark.
- Changed the logo to match the new commit.
- Removed the attribute builder and pop outs theming.
- Updated sf2e.mjs as it wasn't changing between styles.

**Comment by LiftVortex** [comment]
I've completely rewritten the dark.css stylesheet to be inline with the style and formatting of light.css.
I've also removed asset that are no longer used and renamed the remaining ones.

File: module.json [hunk 1]
```diff
@@ -37,8 +37,13 @@
             "flags": {}
         }
     ],
+    "styles": [
```
Comment (stwlam) [inline module.json:40]: Don't include these

File: module.json [hunk 2]
```diff
@@ -37,8 +37,13 @@
...
+        "styles/dark.css", 
+        "styles/light.css"
+    ],
     "scripts": [
-        "sf2e.mjs"
+        "sf2e.mjs", 
+        "scripts/sf2e-Overrides.js"
```
Comment (stwlam) [inline module.json:46]: ja dom alterations are going to be too fragile for something like this

File: sf2e.mjs [hunk 3]
```diff
@@ -2,7 +2,10 @@ const MODULE_ID = "starfinder-field-test-for-pf2e";
 
 function applyTheme(themeId) {
   const headEl = document.head;
-  headEl.querySelector(`link[href*=${MODULE_ID}]`)?.remove();
+
```
Comment (stwlam) [inline sf2e.mjs:5]: This isn't necessary

File: sf2e.mjs [hunk 4]
```diff
@@ -25,10 +28,12 @@ Hooks.once("init", () => {
       choices: {
         none: "SF2E.SETTINGS.Theme.None",
         light: "SF2E.SETTINGS.Theme.Light",
+        dark: "SF2E.SETTINGS.Theme.Dark", // Added dark theme
         // "high-contrast": "SF2E.SETTINGS.Theme.HighConstrast",
       },
       initial: "none",
     }),
+    default: "none",
```
Comment (stwlam) [inline sf2e.mjs:36]: This `initial` is setting the default

File: module.json (hunk 1 above)
Comment (LiftVortex) [reply module.json]: I've now removed these.

File: module.json (hunk 2 above)
Comment (LiftVortex) [reply module.json]: I've now removed this file.

File: sf2e.mjs (hunk 3 above)
Comment (LiftVortex) [reply sf2e.mjs]: I've reverted this to how it was previously.

File: sf2e.mjs (hunk 4 above)
Comment (LiftVortex) [reply sf2e.mjs]: I've removed this setting.

**Comment by LiftVortex** [comment]
@stwlam thanks for the comments.
I've updated the fork to resolve these issues.
Apologies, I only started learning CSS and JavaScript for this project, still got a lot of learning to do. Thanks to your comments I now understand why I was having issues previously!

File: module.json [hunk 5]
```diff
@@ -751,4 +751,4 @@
             }
         }
     }
-}
+}
```
Comment (TikaelSol) [inline module.json:754]:
```suggestion
}

```

File: sf2e.mjs [hunk 6]
```diff
@@ -37,4 +38,4 @@ Hooks.once("init", () => {
 
 Hooks.once("setup", () => {
   applyTheme(game.settings.get(MODULE_ID, "theme"));
-});
+});
```
Comment (TikaelSol) [inline sf2e.mjs:41]:
```suggestion
});

```

File: styles/dark.css [hunk 7]
```diff
@@ -0,0 +1,771 @@
...
+          }
+        }
+      }
+
+      img.logo {
+        display: none;
+      }
+    }
+  }
+}
```
Comment (TikaelSol) [inline styles/dark.css:771]:
```suggestion
}

```

**Comment by LiftVortex** [comment]
@TikaelSol @stwlam , I've updated the fork to resolve the conflicts with the main branch.
I've also fixed some style bugs I've found when testing.
Sorry about the line endings, I didn't notice that GitHub Desktop automatically changes them to CRLF when you commit, I've fixed that now.

File: styles/dark.css [hunk 8]
```diff
@@ -0,0 +1,857 @@
...
```
Comment (TikaelSol) [inline styles/dark.css:857]:
```suggestion
}

```

Yeah, these are a pain🙃

**Pull request merged**